float32 steering_wheel_angle
float32 head_pan
float32 head_tilt
uint32 driving_counter
bool connection_loss
